filebeat/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM elastic/filebeat:7.17.5
+
+COPY dist/filebeat_linux_amd64_v1/beat /usr/share/filebeat/filebeat
```

Comment:
why we add this file?

Reply:
it's used to build a docker image for filebeat